### ISSUE TITLE
MBP 15,2 security lag

### ISSUE THREAD

**Issue by egore24** [the-darkvoid/XPS9360-macOS#96]
Hello DV. Thanks for the excellent work! 

I have been working with using a SMBIOS for MBP 15,2 and have run your script to disable TouchID daemons, but there is still a significant lag when a system level (or third party app) password is required to be entered. I can confirm this is isolated to the MBP 15,2 profile as with a MBP 14,1 the lag is not present. Any ideas on how to eliminate the security prompts lag in 15,2? Thanks much.

**Comment by the-darkvoid** [comment]
Test the following and confirm which works:

    sudo bioutil -s -u 0

It will generate an error because its unable to communicate with the security chip.
Reboot.

If that doesn't work:

    sudo launchctl remove -w /System/Library/LaunchDaemons/com.apple.biometrickitd.plist
    sudo launchctl remove -w /System/Library/LaunchDaemons/com.apple.biokitaggdd.plist

Reboot.

If that doesn't work:

	sudo rm /System/Library/LaunchDaemons/com.apple.biometrickitd.plist
	sudo rm /System/Library/LaunchDaemons/com.apple.biokitaggdd.plist

Reboot.

**Comment by egore24** [comment]
Thanks for the quick reply! It appears the last one listed:

If that doesn't work:

sudo rm /System/Library/LaunchDaemons/com.apple.biometrickitd.plist
sudo rm /System/Library/LaunchDaemons/com.apple.biokitaggdd.plist
Reboot.

Is the one that works. Thanks again!

**Comment by the-darkvoid** [comment]
Thank you for testing that out, I'll update the XPS9360.sh script to reflect this.

In case you are curious, this essentially disables the biometric service daemons in macOS.

Because they are no longer running the GUI does not spend time looking for biometric devices through these services, but rather immediately fails over to password only authentication.

**Comment by egore24** [comment]
Thank you for the explanation. It's good to know the reason behind.

Additionally, I have tested keeping bluetooth alive and found that moving BrcmFirmwareRepo.kext and BrcmPatchRAM2.kext from L/E to EFI/Clover/Kexts/Other maintains its availability after sleep.

Also, I have changed darkwake to a "4" setting. This illuminates the screen when the lid is opened. The current repo darkwake setting leaves a black screen when the lid is opened. A keypress is then needed to illuminate the screen.

You might want to implement these changes as well to the next repo commit. Thanks for everything!
